context/app/api/vitessce_confs/base_confs.py

```diff
@@ -177,39 +177,33 @@ def _setup_scatterplot_view_config(self, vc, dataset):
 
 class SPRMViewConfBuilder(ImagePyramidViewConfBuilder):
 
-    def _get_sprm_image(self):
-        image_file_regex = f"{self._imaging_path}/{self._base_name}.ome.tiff?"
+    def _make_full_image_path(self):
+        return f"{self._imaging_path}/{self._image_name}.ome.tiff?"
+
+    def _check_sprm_image(self, path_regex):
         file_paths_found = self._get_file_paths()
-        found_image_files = get_matches(file_paths_found, image_file_regex)
+        found_image_files = get_matches(file_paths_found, path_regex)
         if len(found_image_files) != 1:
             message = f'Found {len(found_image_files)} image files for SPRM uuid "{self._uuid}".'
             raise FileNotFoundError(message)
         found_image_file = found_image_files[0]
         return found_image_file
 
-    def _add_sprm_image_to_view_conf(self, found_image_file, vc, dataset):
+    def _get_sprm_ometiff_wrapper(self, found_image_file):
```

Comment:
```suggestion
    def _get_ometiff_wrapper(self, found_image_file):
```

Is there anything SPRM specific about this? Or are there other OmeTiffWrapper that we need to distinguish this from?